app/src/main/java/com/nimbl3/di/modules/GlideModule.kt

```diff
@@ -2,6 +2,8 @@ package com.nimbl3.di.modules
 
 import com.bumptech.glide.annotation.GlideModule
 import com.bumptech.glide.module.AppGlideModule
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ApplicationComponent
```

Comment:
How about these imports? I don't see their usages 🤔

Reply:
oh, nice catch, I forget to remove, have been testing it 😅